### PR TITLE
Big refactor on sermons logic/page

### PR DESCRIPTION
**Pull request by xumaple**

### CODE REVIEW

File: frontend/src/components/page-watch/index/sermons.js [hunk 1]
```diff
@@ -3,127 +3,132 @@ import { Link, navigate } from "gatsby";
...
+        {!isLast ? (
+          <Link
+            to={`${nextPage}#sermons-list-paged`}
+            rel="next"
+            className="font-roboto text-Accent-500 no-underline"
+          >
+            &gt;
+          </Link>
+        ) : (
+          <span className="font-roboto opacity-0 text-Shades-0">&lt;</span>
```
Comment (thedomcab) [inline frontend/src/components/page-watch/index/sermons.js:105]:
```suggestion
          <span className="font-roboto opacity-0 text-Shades-0">&gt;</span>
```

File: frontend/src/page-generation/create-page.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+const sh = require("shelljs");
```
Comment (thedomcab) [inline frontend/src/page-generation/create-page.js:1]: Why is this hear? it seems not being used.

File: frontend/src/page-generation/sermon-pages.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
+const hash = require("crypto-js/sha256");
```
Comment (thedomcab) [inline frontend/src/page-generation/sermon-pages.js:1]: it seems discontinued and advise to use node crypto. https://www.npmjs.com/package/crypto-js

**Comment by thedomcab** [comment]
This is invalid UX
image
We should be able to change to a different speaker in this scenario

We should be able to change to different series for this scenario
image

Should be able to change to a different book in this scenario:
image

Should not shorten the list of topics in this scenario:
image

File: frontend/src/page-generation/sermon-pages.js (hunk 3 above)
Comment (xumaple) [reply frontend/src/page-generation/sermon-pages.js:1]: Yes, but we want to still be able to support older node servers. Especially if we end up migrating to a custom server to serve gatsby, we'll need to support node <= v18

Plus thisnis just being used for a simple hash algorithm. There's no cybersecurity involved in what we're doing here, hence no vulnerabilities.

**Review by thedomcab — APPROVED** [review]

**Pull request merged**